### PR TITLE
Feat(#374):-Update feedback-spa type as per feedback-service

### PR DESCRIPTION
**Pull request by anjneeksharma**
# Closes/Fixes/Resolves

### 

# Explain the feature/fix

Update feedback-spa type as we updated feedback-service type to communicate with user-service to fetch its createdBy and UpdatedBy.

## Does this PR introduce a breaking change

No 

## Screenshots

<details>
<summary>View Screenshots</summary>

image

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by sayak-sarkar** [comment]
@anjneeksharma Please remove the console logs and rebase the branch.

File: packages/api-gateway-service/package-lock.json [hunk 1]
```diff
@@ -15030,8 +15030,7 @@
               "version": "2.1.1",
```
Comment (riginoommen) [inline packages/api-gateway-service/package-lock.json:15030]: @anjneeksharma - Can you please check out this file becuse this is not related to feedback schema changes

Comment (anjneeksharma) [reply packages/api-gateway-service/package-lock.json:15030]: yes while rebasing it got some update on that
